AUR and more! (#19)
vignetteapp/docs: docs/install/install-aur.md

# Installing Vignette using AUR

> If you wish to install Vignette on your Arch Linux computer, you may use any AUR helper of you choice. The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/). In the example below, I will be using yay.

Reviewer: I still stand on my suggestion here

```suggestion
If you wish to install Vignette on your Linux computer, you can use any AUR helper of your choice.

The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/).

Here I'll be using `yay`.
```

Author: stupid typo on "you" lol. as for the rest, sure, have it your way

## Installation process

> In order to install Vignette using yay, please run the following command: 

Reviewer: I'd say do it like this. It really sounds like you're begging them to run the command... 

How does it sound without?
```suggestion
In order to install Vignette using yay, run the following command:

```

Author: fine by me

`yay vignette-git`

> If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. In this example, that package is number `1`.

Reviewer: hmm like this?

```suggestion
If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. Here, that package is number `1`.
```

Author: "in this example" phrasing is consistent with the rest of the page

![](/img/install/yay-terminal-window.webp)

:::note 

You may be required to type your password into the terminal when prompted.

:::

## Welcome to Vignette!

> Please visit the UI Overview page [here](/client/overview.md) or from the sidebar in order to make yourself more familiar with the interface and features of the app. Thank you for chosing Vignette!

Reviewer: they aren't required to read it if they wanna go on their own.

```suggestion
You can now visit the UI Overview page [here](/client/overview.md) in order to make yourself more familiar with the interface and features of the app.

Thank you for installing Vignette!
```

Author: please sounds more friendly, and the sidebar thing is a good backup

Reviewer: "you can now do this" sounds way better than "please do this". You're not begging them

Author: if you cant install an AUR package you most likely cant figure the UI out yourself, so being more pushy seems right

Reviewer: i mean... people don't read this because they don't know how to do it. Maybe they don't know what the package name is, or if there even is an AUR package.

Author: yay --query is a thing people use

Reviewer: i didn't even know it existed. Well... this is getting off-topic. Since we can't decide, let's get a third opinion

Author: @sr229 could you please weigh in?

# Installing Vignette using AUR

> If you wish to install Vignette on your Linux computer, you can use any AUR helper of your choice.

Reviewer 2: ```suggestion
If you wish to install Vignette on your Linux computer, you can use any AUR helper of your choice.

If you prefer to install Vignette on Linux, we provide an official AUR package. For this guide, we will be using `yay`.
```

> The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/).

Reviewer 2: Preferrably put it on the same sentence where I put the suggestion in.

> Here I'll be using `yay`.

Reviewer 2: Remove this, and please don't do first-person, this is a professional documentation.

## Installation process

In order to install Vignette using yay, run the following command:

`yay vignette-git`

If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. In this example, that package is number `1`.

> ![](/img/install/yay-terminal-window.webp)

Reviewer 2: Add an alt text here just in case we have people who has failure loading the image

:::note 

> You may be required to type your password into the terminal when prompted.

Reviewer 2: ```suggestion
When prompted for your password for sudo, enter it to proceed with the installation.
```

:::

## Welcome to Vignette!

> Please visit the UI Overview page [here](/client/overview.md) or from the sidebar in order to make yourself more familiar with the interface and features of the app. Thank you for chosing Vignette!

Reviewer 2: ```suggestion
Don't forget to visit the [UI Overview](/client/overview.md) once your installation has completed. Alternatively, you can use the sidebar to get familiar with the interface of the application.
```
